packages/turf-along/test.js

```diff
@@ -17,15 +20,31 @@ test('turf-along', t => {
     const pt7 = along(line, 100, options);
     const pt8 = along(line.geometry, 0, options);
     const fc = featureCollection([pt1, pt2, pt3, pt4, pt5, pt6, pt7, pt8]);
-
+    
     fc.features.forEach(f => {
         t.ok(f);
         t.equal(f.type, 'Feature');
         t.equal(f.geometry.type, 'Point');
     });
     t.equal(fc.features.length, 8);
-    t.equal(fc.features[7].geometry.coordinates[0], pt8.geometry.coordinates[0]);
-    t.equal(fc.features[7].geometry.coordinates[1], pt8.geometry.coordinates[1]);
+  
+    t.equal(isPointOnLine(fc.features[5].geometry.coordinates, line), true);
+    t.equal(isPointOnLine(fc.features[6].geometry.coordinates, line), true);
+    t.equal(isPointOnLine(fc.features[7].geometry.coordinates, line), true);
+    
+    if (process.env.REGEN) write.sync(path.join(__dirname, 'test', 'fixtures', 'dc-points.geojson'), fc);
+    t.deepEqual(fc, load.sync(path.join(__dirname, 'test', 'fixtures', 'dc-points.geojson')));
 
     t.end();
 });
+
+const longLine = load.sync(path.join(__dirname, 'test', 'fixtures', 'long-line.geojson'));
+
+test('turf-along-long', t => {
+  const pt1 = along(longLine.features[0], 200, {units: 'miles'});
```

Comment:
If I'm right you draw a point on the line and then you want the calculated point along the line to match the one you set, correct?
If so I'd write it like this, to make it a little more obvious:
```js

test('turf-along-long', t => {
  const features = load.sync(path.join(__dirname, 'test', 'fixtures', 'long-line.geojson'));
  const longLine = features.features[0];
  const pointOnLongLine = longLine.features[1];

  const pt1 = along(longLine, 200, {units: 'miles'});
  t.deepEqual(pt1, pointOnLongLine);

  t.end();
});